controllers/tools/applications/views/applications.njk

```diff
@@ -46,7 +46,7 @@
             </h1>
 
             <p>
-                <strong>Date range</strong>: {{ formatDate(dateRange.start) }}–{{ formatDate(dateRange.end) }}
+                {{ formatDate(dateRange.start) }}–{{ formatDate(dateRange.end) }}
```

Comment:
This was just for consistency with the other tools (which don't have this "Date range:" prefix here)